heron/shell/src/python/main.py

```diff
@@ -21,25 +21,9 @@
 from tornado.httpclient import AsyncHTTPClient
 from tornado.options import define, options, parse_command_line
 
-from heron.shell.src.python import handlers
+from heron.shell.src.python import utils
 
-AsyncHTTPClient.configure(None, defaults=dict(request_timeout=120.0))
-app = tornado.web.Application([
-    (r"^/jmap/([0-9]+$)", handlers.JmapHandler),
-    (r"^/histo/([0-9]+$)", handlers.MemoryHistogramHandler),
-    (r"^/pmap/([0-9]+$)", handlers.PmapHandler),
-    (r"^/jstack/([0-9]+$)", handlers.JstackHandler),
-    (r"^/pid/(.*)", handlers.PidHandler),
-    (r"^/browse/(.*)", handlers.BrowseHandler),
-    (r"^/file/(.*)", handlers.FileHandler),
-    (r"^/filedata/(.*)", handlers.FileDataHandler),
-    (r"^/filestats/(.*)", handlers.FileStatsHandler),
-    (r"^/download/(.*)", handlers.DownloadHandler),
-    (r"^/killexecutor", handlers.KillExecutorHandler),
-])
-
-
-if __name__ == '__main__':
+def run(extra_handlers=[]):
```

Comment:
Could we instead allow the caller to pass the entire list of handlers, so they could either remove or add to the defaults?
```
def run(handlers=utils.default_url_to_handlers)
```